.husky/post-commit

```diff
@@ -1,4 +1,7 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn install
+## This hook was disabled in https://github.com/blockprotocol/blockprotocol/pull/184
+## @todo: Revert when whe cleanup `prepare` script in `package.json`
```

Comment:
small typo
```suggestion
## @todo: Revert when we cleanup `prepare` script in `package.json`
```

Reply:
Thanks for spotting this typo @teenoh! I’ve added the fix to https://github.com/blockprotocol/blockprotocol/pull/178/files#diff-b6aebccfc7f539deea6c7023d96b327318a788d5193a403074d8e4568f753413